zio-cli/js/src/main/scala/zio/cli/oauth2/OAuth2PlatformSpecific.scala

```diff
@@ -0,0 +1,13 @@
+package zio
+package cli
+package oauth2
+
+object OAuth2PlatformSpecific {
+  def validate(
+    provider: OAuth2Provider,
+    scope: List[String],
+    auxiliaryOptions: Options[OAuth2AuxiliaryOptions],
+    args: List[String],
+    conf: CliConfig
+  ): IO[ValidationError, (List[String], OAuth2Token)] = ???
```

Comment:
To provide a good error message, you should use:

```suggestion
  ): IO[ValidationError, (List[String], OAuth2Token)] = ZIO.dieMessage("OAuth2 support is not currently implemented for Scala.js. If you are interested in adding support, please open a pull request at https://github.com/zio/zio-cli.")
```

Reply:
Use this suggestion, thanks!